src/frontend/src/pages/AdminToolsPage/ProjectsConfig/LinkTypeFormModal.tsx

```diff
@@ -0,0 +1,146 @@
+        toast.error(error.message);
+      }
+    }
+    handleClose();
+  };
+  const tooltipMessage = (
+    <Typography sx={{ fontSize: 14 }}>
+      Click to view possible icon names. For names with multiple words, seperate them with an _. AttachMoney = attach_money
+    </Typography>
+  );
+  return (
+    <NERFormModal
+      open={showModal}
+      onHide={handleClose}
+      title={!!defaultValues ? 'Edit LinkType' : 'Create LinkType'}
+      reset={() => reset({ name: '' })}
+      handleUseFormSubmit={handleSubmit}
+      onFormSubmit={onFormSubmit}
+      formId={!!defaultValues ? 'edit-LinkType-form' : 'create-LinkType-form'}
+      showCloseButton
+    >
+      <Grid container spacing={2} alignItems="flex-start">
+        <Grid item xs={6}>
+          <FormControl fullWidth>
+            <FormLabel>LinkType Name</FormLabel>
+            <ReactHookTextField name="name" control={control} />
+            <FormHelperText error>{errors.name?.message}</FormHelperText>
+          </FormControl>
+        </Grid>
+        <Grid item xs={6}>
+          <FormControl fullWidth>
+            <FormLabel sx={{ '&.Mui-focused': { color: theme.palette.text.secondary } }}>Required</FormLabel>
+            <Controller
+              name="required"
+              control={control}
+              render={({ field }) => <Switch {...field} checked={field.value} />}
+            />
+            <FormHelperText error>{errors.required?.message}</FormHelperText>
+          </FormControl>
+        </Grid>
+        <Grid item xs={6}>
+          <FormControl fullWidth>
+            <Box style={{ display: 'flex', verticalAlign: 'middle', alignItems: 'center' }}>
+              <FormLabel>Icon Name</FormLabel>
+              <Tooltip title={tooltipMessage} placement="right">
+                <a
+                  href="https://mui.com/components/material-icons/"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  style={{ textDecoration: 'none' }}
+                >
+                  <HelpIcon style={{ marginBottom: '-0.2em', fontSize: 'medium', marginLeft: '5px', color: 'lightgray' }} />
+                </a>
+              </Tooltip>
+            </Box>
+            <ReactHookTextField name="iconName" control={control} />
+            <FormHelperText error>{errors.iconName?.message}</FormHelperText>
+          </FormControl>
+        </Grid>
+        <Grid item xs={6}>
+          <FormControl fullWidth>
+            <FormLabel>Icon Preview</FormLabel>
+            <Box
+              sx={{
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+                border: 1,
+                borderColor: 'rgb(118, 118, 118)',
+                padding: '8px',
+                borderRadius: '6px',
+                marginTop: '1px',
+                marginLeft: '0.25em',
+                height: '56px',
+                width: '84px'
```

Comment:
change width to 96px and remove marginLeft to make this input more in line with the other inputs